app/gateway/kong-manager.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Kong Manager"
+content_type: reference
+layout: reference
+
+products:
+    - gateway
+
+min_version:
+  gateway: '3.4'
+
+description: Kong Manager is the graphical user interface (GUI) for {{site.base_gateway}}.
+
+faqs:
+  - q: I can't access Kong Manager. How do I fix a Kong Manager URL that doesn’t resolve?
+    a: |
+      Most likely, the port wasn't exposed during installation. Install a new instance and map port `8002` during installation.
+      
+      For example, with a Docker install:
+
+      ```
+      -p 127.0.0.1:8002:8002
+      ```
+
+related_resources:
+  - text: Enable Basic Auth for Kong Manager
+    url: /how-to/enable-basic-auth-on-kong-manager/
+  - text: Enable OIDC for Kong Manager
+    url: /how-to/enable-oidc-for-kong-manager/
+  - text: Set up authenticated group mapping in Kong Manager with OIDC
+    url: /how-to/oidc-authenticated-group-mapping/
+  - text: Configure LDAP with Kong Manager
+    url: /how-to/configure-ldap-with-kong-manager/
+  - text: Configuring Kong Manager to Send Email
+    url: /how-to/configure-kong-manager-email/
+---
+
+@todo
+
+Pull content from https://docs.konghq.com/gateway/latest/kong-manager/
+
+If you're running {{site.base_gateway}} with a database (either in traditional
+or hybrid mode), you can enable {{site.base_gateway}}'s graphical user interface
+(GUI), Kong Manager.
+
+## Enable Kong Manager
+
+To enable Kong Manager, set the [`KONG_ADMIN_GUI_PATH`](/gateway/configuration/#admin_gui_path) and [`KONG_ADMIN_GUI_URL`](/gateway/configuration/#admin_gui_url) properties in the ([`kong.conf`](/gateway/configuration/)) configuration file to the DNS or IP address of your system, then reload {{site.base_gateway}} with `kong reload` for the setting to take effect.
+
+If you're running {{site.base_gateway}} in Docker, you'd use the following:
+
+```bash
+docker exec -i $KONG_CONTAINER_ID /bin/sh -c "export KONG_ADMIN_GUI_PATH='/'; export KONG_ADMIN_GUI_URL='http://localhost:8002/manager'; kong reload; exit"
```

Comment:
```suggestion
docker exec -i $KONG_CONTAINER_ID /bin/sh -c \
"export KONG_ADMIN_GUI_PATH='/'; \
export KONG_ADMIN_GUI_URL='http://localhost:8002/manager'; \
kong reload; \
exit"
```
So that we don't have a scrolling codeblock.